tests/unit-tests/dds/DCPS/Statistics.cpp

```diff
@@ -0,0 +1,23 @@
+#include <dds/DCPS/Statistics.h>
+
```

Comment:
```suggestion
```

Reply:
This is a common pattern in unit tests - the thing we're testing is its own "group"